k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-pod.yaml

```diff
@@ -120,8 +120,8 @@ spec:
             - name: UDS_CLOUD_CONTAINER_NAME
               value: {{ .Values.cortxserver.name }}
           ports:
-          - containerPort: 8000
-          - containerPort: 8443
+          - containerPort: 22751
```

Comment:
We don't have to do it now, but if we add a name to these ports, then that name can be referenced in the Service's corresponding targetPort field in `cortx-io-svc.yaml` instead of needing to manually keep the ports in sync (so that way the names are only written once and the service automatically tracks the containerPort changes in the Pod spec).

Reply:
Good idea. I'll go ahead and add this, I prefer it to manually syncing them.